React/todoApp/src/components/Form.module.css

```
@@ -0,0 +1,55 @@
+.form {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  gap: 12px;
+  width: 100%;
+}
+
+.label {
+  display: contents; 
+}
+
+.input {
+  width: 100%;
+  height: 50px;
+  border: 1px solid var(--border);
+  border-radius: var(--r-md);
+  background: var(--panel);
+  color: var(--text);
```

Using variables in CSS is a great way to encapsulate styles within certain domains. 
My suggestions here would be to use meaningful names for the styles

Your application can be divided into `primary` and `secondary` color groups where `primary` being the main color and `secondary` being the accent color. 


```suggestion
  border: 1px solid var(--primary-broder-color);
  border-radius: var(--radius-md);
  background: var(--primary-bg);
  color: var(--primary-text);
```